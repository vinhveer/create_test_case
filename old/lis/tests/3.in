79
154 304 930 304 771 218 954 957 781 394 452 182 348 359 357 108 719 58 627 398 533 634 843 551 453 311 515 89 451 803 143 573 808 892 22 891 852 714 4 529 340 935 198 298 339 389 398 309 53 502 824 828 9 619 970 120 932 764 737 580 600 534 998 835 200 267 11 803 512 190 609 22 137 647 619 930 727 227 823 
61
42 459 556 452 772 870 568 539 111 846 565 31 442 296 507 508 594 413 698 999 703 754 403 981 284 494 195 332 396 515 815 114 36 159 972 373 43 608 134 457 586 719 432 432 121 508 893 185 680 210 228 349 47 930 159 429 382 205 798 831 233 
