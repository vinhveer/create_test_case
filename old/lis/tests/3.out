3
452 515 930 
